lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -923,112 +749,17 @@ spec = do
             in
                 x' === x .&&. show x' === show x
 
--- Golden tests files are generated automatically on first run. On later runs
--- we check that the format stays the same. The golden files should be tracked
--- in git.
---
--- Example:
--- >>> roundtripAndGolden $ Proxy @ Wallet
---
--- ...will compare @ToJSON@ of @Wallet@ against `Wallet.json`. It may either
--- match and succeed, or fail and write `Wallet.faulty.json` to disk with the
--- new format. Faulty golden files should /not/ be commited.
---
--- The directory `test/data/Cardano/Wallet/Api` is used.
-jsonRoundtripAndGolden
```

Comment:
Extracted in `test-utils` to be available from multiple packages.

Reply:
good move